src/models/controller.ts

```diff
@@ -0,0 +1,94 @@
+import express from 'express'
+const router = express.Router()
+import { v4 as uuid } from 'uuid'
+import { Request, Response, NextFunction } from 'express'
+import UserService from '../models/user'
+
+interface RequestBody {
```

Comment:
For each controllers you need create separate interface for req.body

Reply:
Done